portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro

```diff
@@ -41,6 +37,19 @@ definition {
 		WaitForPageLoad();
 	}
 
+	macro completeWizardConfiguration {
+		if (IsElementPresent(key_text = "Next", locator1 = "Button#ANY")) {
```

Comment:
Hey @joshitagaki, I think you need to put a `while` loop here so that Poshi clicks on all Next buttons that are present. You don't know which step of the wizard this macro will be used in, so you need to cover all cases. What do you think about switching to:

`while (IsElementPresent(key_text = "Next", locator1 = "Button#ANY")) {`

I think that way whenever the Next button appears it will be clicked until it is not present on the screen

and then there would be the click on the Finish button, I think there is no need to have the conditional here since whenever this macro is used, so in the end it will be necessary and expected that there will be a click on Finish

Reply:
Yea you're right. Meant to do a loop haha. Thanks for pointing that out